web/src/app/navbar/navbar.component.html

```diff
@@ -19,7 +19,7 @@
     </ul>
 
     <div class="navbar-nav" *ngIf="isLoggedIn()">
-      <a class="nav-item nav-link" *ngIf="getUserEmail() != null"> {{ getUserEmail() }} </a>
+      <a class="nav-item nav-link" *ngIf="getUserEmail() != null" href="/user-profile"> {{ getUserEmail() }} </a>
```

Comment:
getUserEmail?

Reply:
It is from firebase to display it on the navbar